docker-compose.yml

```diff
@@ -52,6 +58,10 @@ mysql-data:
 
 mysql:
   build: ./docker/images/mysql
+  entrypoint:
+    - docker-entrypoint.sh
```

Comment:
Docs for ``docker-compose pull --help``:

```
Pulls images for services.

Usage: pull [options] [SERVICE...]

Options:
    --ignore-pull-failures  Pull what it can and ignores images with pull failures.
```

I don't see ``--no-cache`` as an option, @safwanrahman.  I'm adding "Update stale docker images" to my future docker debugging docs.